package.json

```diff
@@ -592,37 +592,45 @@
                     "command": "makefile.setLaunchConfiguration",
                     "when": "makefile:localDebugFeature"
                 },
+                {
+                    "command": "makefile.outline.configure",
+                    "when": "never"
+                },
```

Comment:
All the changes with "never" will cause the "outline" commands counterparts to not be visible in the palette. The palette was polluted with all these "outline" commands which make sense for the left side bar UI not for the palette.
I saw in the CMake Tools commands being written with "never" condition and I didn't understand why that was needed, thinking that removing the whole block would be the same as having it with "never". It's clear now how to make a command not show up in the palette.